packages/clerk-js/src/ui/styledSystem/animations.ts

```diff
@@ -84,7 +84,7 @@ const notificationAnimation = keyframes`
 const outAnimation = keyframes`
   0% {
     opacity:1;
-    translateY(0px);
+    translateY: (0px);
```

Comment:
ouch !! NIce finding, could you open an independent PR with this fix ?

Reply:
Sure, will do in a bit!

Reply:
https://github.com/clerk/javascript/pull/4658#discussion_r1858847037